Dependency issue with Angular 17.3.6 and failed to compile

Angular and NodeJs version
````
Angular CLI: 17.3.6
Node: 18.20.4
Package Manager: npm 10.7.0
OS: linux x64
````

Installed module version
` @worktile/gantt@18.1.0-next.0`

Errors during compilation:

> ./node_modules/@worktile/gantt/fesm2022/worktile-gantt.mjs:18:0-71 - Error: Module not found: Error: Can't resolve 'ngx-gantt' in '/var/www/html/PROJECT/node_modules/@worktile/gantt/fesm2022'

> Error: node_modules/@worktile/gantt/components/main/gantt-main.component.d.ts:4:39 - error TS2307: Cannot find module 'ngx-gantt' or its corresponding type declarations.
import { NgxGanttRootComponent } from 'ngx-gantt';

> Error: node_modules/date-fns/intlFormatDistance.d.ts:6:16 - error TS2724: 'Intl' has no exported member named 'RelativeTimeFormatOptions'. Did you mean 'DateTimeFormatOptions'?
extends Intl.RelativeTimeFormatOptions,

--- a/packages/gantt/src/gantt.module.ts
+++ b/packages/gantt/src/gantt.module.ts
@@ -24,7 +24,6 @@ import { NgxGanttTableColumnComponent } from './table/gantt-column.component';
 import { NgxGanttTableComponent } from './table/gantt-table.component';
 import { GanttScrollbarComponent } from './components/scrollbar/scrollbar.component';
 import { i18nLocaleProvides } from './i18n';
-import { setDefaultTimeZone } from 'ngx-gantt';
 
 @NgModule({
     imports: [

